art/preprocessing/standardisation_mean_std/standardisation_mean_std_tensorflow.py

```diff
@@ -49,19 +54,21 @@ def __init__(
         :param std: Standard Deviation.
         """
         super().__init__(is_fitted=True, apply_fit=apply_fit, apply_predict=apply_predict)
-        self.mean = mean
-        self.std = std
+        self.mean = np.asarray(mean, dtype=ART_NUMPY_DTYPE)
+        self.std = np.asarray(std, dtype=ART_NUMPY_DTYPE)
         self._check_params()
 
     def forward(self, x: "tf.Tensor", y: Optional["tf.Tensor"] = None) -> Tuple["tf.Tensor", Optional["tf.Tensor"]]:
         """
         Apply standardisation with mean and standard deviation to input `x`.
+
+        :param x: Input samples to standardise of shapes `NCHW`, `NHWC`, `NCFHW` or `NFHWC`.
```

Comment:
Does this new code also work with tabular data?

Reply:
This docstring addition is legacy as I copied it from its Numpy counterpart. I don't see why standardization should not work for tabular data. Thus we could improve the documentation I assume. What do you think?

The `broadcastable_mean_std` can handle the tabular case already.

Reply:
As discussed in our chat, standardization works for different input shapes as well. Therefore I have updated docstrings accordingly.